<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndUserRegistrationStepWidget</class>
 <widget class="QWidget" name="EndUserRegistrationStepWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>&amp;First Name</string>
       </property>
       <property name="buddy">
        <cstring>firstNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="firstNameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="middleNameLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Middle Initial</string>
       </property>
       <property name="buddy">
        <cstring>middleNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="middleNameLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lastNameLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Last Name</string>
       </property>
       <property name="buddy">
        <cstring>lastNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lastNameLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="streetAddressLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Site Street &amp;Address</string>
       </property>
       <property name="buddy">
        <cstring>addressLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="addressLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="cityLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Site &amp;City</string>
       </property>
       <property name="buddy">
        <cstring>cityLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="cityLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="zipCodeLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;ZIP Code</string>
       </property>
       <property name="buddy">
        <cstring>zipCodeLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="zipCodeLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="stateCountryLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Site &amp;Country</string>
       </property>
       <property name="buddy">
        <cstring>countryLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="countryLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>United State of America</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="emailAddressLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>End user &amp;E-mail Address</string>
       </property>
       <property name="buddy">
        <cstring>emailAddressLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="emailAddressLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="phoneNumberLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>End user &amp;Phone Number</string>
       </property>
       <property name="buddy">
        <cstring>phoneNumberLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="phoneNumberLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="dealerIdLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Dealer &amp;ID</string>
       </property>
       <property name="buddy">
        <cstring>dealerIDLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="dealerIDLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>&amp;Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
